serpextract/serpextract.py

```diff
@@ -558,11 +544,7 @@ def get_parser(referring_url):
     # search engines, yahoo and yahoo images
     if u'{}{}'.format(domain, path) in engines:
         engine_key = u'{}{}'.format(domain, path)
-    elif u'{}{}'.format(lossy_domain, path) in engines:
-        engine_key = u'{}{}'.format(lossy_domain, path)
-    elif lossy_domain in engines:
-        engine_key = lossy_domain
-    elif domain not in engines:
+    elif domain not in engines and domain.strip('www.') not in engines:
```

Comment:
Ah, this is one of my favorite super common bugs.

```py
In [21]: domain = 'www.wegotbugs.com'

In [22]: domain.strip('www.')
Out[22]: 'egotbugs.com'
```

You want `domain[4:] if domain.startswith('www.') else domain`.